mqtt-sn-sub.c

```diff
@@ -131,7 +135,11 @@ static void parse_opts(int argc, char** argv)
             break;
 
         case 'v':
-            verbose = TRUE;
+            verbose = verbose == 0 ? 1 : verbose;
```

Comment:
Exactly, to prevent decreasing verbose level so that -V has precedence over -v.
OK, I will submit new PR.